echopop/survey.py

```diff
@@ -10,7 +10,7 @@
     stratified_summary,
 )
 from .core import DATA_STRUCTURE
-from .utils import load as el, message as em
+from .utils import batch_load as ebl, load as el, message as em
```

Comment:
I am surprised that black/isort allows this! 🤪

Reply:
Ok, after looking into what each py file have, I would suggest the following:
- rename `batch_load` to `load_nasc` and `load` to `load_survey`
- use the full module name, so `from .utils import load_nasc, load_survey, message` and just have a few more characters in the code when importing functions from these modules.

Reply:
> I am surprised that black/isort allows this! 🤪
I actually believe that `black` recommended this change (I could be misrembering...).

Reply:
Ok! Good to know!